gonomics/vcfBenchmarking/runCallerBenchmarks.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+gunzip -c testdata/1mb.fa.gz > testdata/1mb.fa
+gunzip -c testdata/1mb_mut.fa.gz > testdata/1mb_mut.fa
+gunzip -c testdata/truePos.vcf.gz > testdata/truePos.vcf
+
+samtools faidx testdata/1mb.fa
+samtools faidx testdata/1mb_mut.fa
+
+simulateSam -coverage 1000 -flatErrorRate 0.01 -setSeed 555 testdata/1mb_mut.fa stdout | samtools sort -@4 > testdata/mut100%_1000x.bam
```

Comment:
should this not be $GOBIN/simulateSam, same with callVariants so that they just need it installed? you do specify that it needs to be.